routes/api.php

```diff
@@ -35,3 +50,33 @@
         Route::post('/', StoreUserController::class);
         Route::delete('/{user}', DeleteUserController::class);
     });
+
+Route::prefix('cities')
+    ->middleware([])
+    ->group(static function () {
+        Route::post('/', StoreCityController::class)->name('cities');
+        Route::get('/', ListCitiesController::class);
+        Route::get('/{city}', GetCityController::class)->withTrashed();
+        Route::patch('/{city}', UpdateCityController::class);
+        Route::delete('/{city}', DeleteCityController::class);
```

Comment:
You can use prefix as well here

```suggestion
    Route::prefix('{city}')->group(function () {
            Route::get('/', GetCityController::class);
            Route::patch('/', UpdateCityController::class);
            Route::delete('/', DeleteCityController::class);
        });
```